mmtk/src/api.rs

```diff
@@ -97,6 +105,17 @@ pub extern "C" fn alloc_slow_bump_monotone_immortal(
     unsafe { &mut *(allocator as *mut BumpAllocator<OpenJDK>) }.alloc_slow(size, align, offset)
 }
 
+#[no_mangle]
+pub extern "C" fn is_in_reserved_malloc(obj: ObjectReference) -> bool {
+    if !cfg!(feature = "marksweep") {
```

Comment:
We probably should add an API function to check if an object reference is in mmtk heap. This is internal implementation - we should not expose this and let a binding to deal with it.